hazelcast-sql-core/src/main/java/com/hazelcast/sql/impl/calcite/HazelcastSqlToRelConverter.java

```diff
@@ -206,6 +220,50 @@ private RexNode convertCast(SqlCall call, Blackboard blackboard) {
         return getRexBuilder().makeCast(to, convertedOperand);
     }
 
+    /**
+     * Convert "val BETWEEN lower_bound AND upper_bound" expression to
+     * 1. If ASYMMETRIC : val >= lower_bound AND val =< upper_bound " expression. Default mode.
+     * 2. If SYMMETRIC : (val >= lower_bound AND val =< upper_bound) OR (val <= lower_bound AND val >= upper_bound)
+     * expression
+     */
+    public RexNode convertBetween(SqlCall call, Blackboard blackboard) {
```

Comment:
Could not spot any differences between(!) this and what is in `StandardConvertletTable.convertBetween()`. Are there any?

Reply:
Yes, they are.` StandardConvertletTable#convertBetween` has a signature 
`public RexNode convertBetween(SqlRexContext cx, SqlBetweenOperator op, SqlCall call)`, which doesn't fit for us, because it uses `SqlBetweenOperator` as a second parameter. 
Our `HazelcastBetweenOperator` doesn't extends Calcite's `SqlBetweenOperator`.